apollo-api/src/main/java/com/apollostack/android/PostBody.java

```diff
@@ -0,0 +1,24 @@
+package com.apollostack.android;
+
+import com.apollostack.api.GraphQLOperation;
+import com.apollostack.api.GraphQLQuery;
+
+import javax.annotation.Nullable;
+
+public class PostBody<T extends GraphQLOperation.Variables> {
```

Comment:
Unfortunately Moshi can't handle it the `variables` serialization. I think it's because of this issue: https://github.com/square/moshi/issues/133